templates/web/base/report/new/category.html

```diff
@@ -5,7 +5,11 @@
     <label for='form_category' id="form_category_label">
         [%~ loc('Category') ~%]
     </label>[% =%]
-    <select class="form-control" name='category' id='form_category' data-role='[% c.user.has_body_permission_to('planned_reports') ? 'inspector' : 'user'  %]' data-body='[% c.user.from_body.name %]'>
+    <select class="form-control" name="category" id="form_category"
+    [%~ IF c.user.from_body ~%]
```

Comment:
This will mean the output is `id="form_category"data-role=` (no space), and similar at end, and in other template.